pkg/kubelet/apis/stats/v1alpha1/types.go

```diff
@@ -95,6 +95,9 @@ type PodStats struct {
 	// +patchMergeKey=name
 	// +patchStrategy=merge
 	VolumeStats []VolumeStats `json:"volume,omitempty" patchStrategy:"merge" patchMergeKey:"name"`
+	// Stats pertaining to total usage of filesystem resources on the rootfs used by a pod.
```

Comment:
Suggest `EphemeralStorage reports total usage of filesystem resources on the measured Pod's rootfs`

Reply:
there is no rootfs for pod. Only container has a metric of rootfs. Could you explain what do you mean here?

Reply:
Just trying to clarify the original comment, which says "rootfs used by a pod." Would "rootfs used by the containers in the measured Pod" be more clear?

Reply:
the ephemeral storage at pod level will be the sum of all containers disk usage (including rootfs and logs), plus pod's emptyDir volume (if not backed up by memory) usage

Reply:
So maybe `EphemeralStorage reports the total filesystem usage for the containers and emptyDir volumes in the measured Pod`?

Reply:
I see. changed the comment.